auth/src/main/java/com/firebase/ui/auth/ui/email/WelcomeBackPasswordPrompt.java

```diff
@@ -79,6 +80,10 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.fui_welcome_back_password_prompt_layout);
 
+        if (savedInstanceState != null) {
+            mPendingPassword = savedInstanceState.getString(ExtraConstants.EXTRA_PASSWORD, null);
+        }
```

Comment:
Maybe I'm not fully understanding the idea behind these pending data, but wouldn't it make more sense to keep those around in the `ViewModel`? That seems to be what you're trying to do... 🤷‍♂️

Reply:
Doh. Thank you.  I still have some bad old instincts from the pre-ViewModel dark ages.